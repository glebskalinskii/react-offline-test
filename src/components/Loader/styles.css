.loader {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%); 
}

.loader-text {
  margin-top: 24px;
  font-size: 24px;
  text-align: center;
  color: var(--color-secondary); 
}
