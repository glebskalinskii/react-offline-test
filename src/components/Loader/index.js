import React from 'react';
import IconChargingBulb from '../icons/ChargingBulb';

import './styles.css';

const Loader = () => {
  return <div className="loader">
    <IconChargingBulb />
    <h2 className="loader-text">Charging...</h2>
  </div>
}

export default Loader;
