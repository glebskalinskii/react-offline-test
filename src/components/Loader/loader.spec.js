import React from 'react';
import { shallow } from 'enzyme';
import Loader from './index';

describe('Loader', () => {
  test('renders and matches snapshot', () => {
    const wrapper = shallow(<Loader />);

    expect(wrapper).toMatchSnapshot();
  });
});
