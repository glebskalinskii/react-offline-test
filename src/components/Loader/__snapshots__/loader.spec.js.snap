// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders and matches snapshot 1`] = `
<div
  className="loader"
>
  <IconChargingBulb />
  <h2
    className="loader-text"
  >
    Powering up...
  </h2>
</div>
`;
